recommendations/forms.py

```diff
@@ -26,9 +26,7 @@ class YearForm(forms.Form):
     )
 
     OPTIONS = YEAR_SPANS
-    Years = forms.MultipleChoiceField(
-        widget=forms.CheckboxSelectMultiple, choices=OPTIONS
-    )
+    Years = forms.ChoiceField(widget=forms.RadioSelect, choices=OPTIONS)
```

Comment:
I think that selecting multiple year ranges is actually pretty useful! If you really want the user to stick with the one year span maybe you could have two versions of the year form?

Reply:
for the time being, i'm only going to allow one year range.
i would like to do stuff like that too (hey i want any movie before 1980), but with our deadline and the idea of 'curbing choice overload', we're gonna leave it like this for now.